charts/bpa/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: bpa
 description: The Business Partner Agent allows to manage and exchange master data between organizations.
 type: application
 
-version: 0.3.0-alpha04
+version: 0.3.0-alpha05
 appVersion: 0.1.0-alpha10
```

Comment:
I assume for the keycloak features a newer appVersion ( BPA image) is required , not?

Reply:
I only updated the chart version, wasn't sure if we were ready to tag the BPA code yet.  Would we make appVersion match the GitHub release tag? ex. bpa-0.2.0-alpha03?